src/environment.js

```diff
@@ -1253,6 +1253,9 @@ export class LexicalEnvironment {
   }
 
   evaluate(ast: BabelNode, strictCode: boolean, metadata?: any): Value | Reference {
+    if (this.realm.enableDebugger && this.realm.attachedDebugger && ast.loc) {
```

Comment:
This suggests that a debugger can be attached while debugging is disabled. Is this the case? If so, why?